_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/mentors-open-source-hackillinois/panel_speakers.jpg "panel")
+
+Afterward, hacking began! Students began learning the ropes of open source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it. 
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/mentors-open-source-hackillinois/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from IUPUI and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long term relationships between students and the open source community, and we're super glad that projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [Rustbridge](https://github.com/rustbridge) are still being contributed to and built on.
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/mentors-open-source-hackillinois/rust.jpg "rust")
+
+# Projects
+
+Tons of open source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large open source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These were great because students and mentors could work together to contribute to some of the tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing open source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn). We love these open source projects because they get great opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to just contributing to open source, though. Many students developed their own open source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul) even has 69 stars on Github and was mentioned in a talk at GoogleIO! And it was born right here at HackIllinois.
```

Comment:
> just contributing to open source

only contributing to existing open source projects

 > even has 69 stars on Github

, even has 70+ stars on GitHub as of the time of publication

> GoogleIO

Google I/O
We could also include a link to the talk or the tweet.